src/Symfony/Component/HttpFoundation/StreamedJsonResponse.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+use Generator;
```

Comment:
We don't import classes from the global namespace.

Reply:
Would be interesting if the fabbot could handle this we use the following in php-cs-fixer:

```
'global_namespace_import' => ['import_classes' => false, 'import_constants' => false, 'import_functions' => false],
```

Reply:
Sorry if it's out of topic, but can you please clarify why importing classes from the global namespace is avoided ?

Reply:
@ibousfiha it's just a coding standard. Some prefer to have the usage of `new \DateTime` instead of `new DateTime` so they directly see that the global class is used. When using `new DateTime()` it could also be `use Some\Other\DateTime;`.